packages/orbit-components/src/SkipLink/__tests__/index.test.js

```diff
@@ -1,29 +1,30 @@
 // @flow
 import * as React from "react";
-import { shallow } from "enzyme";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
 
 import SkipLink from "..";
 
 describe("SkipLink", () => {
-  const links = [
-    {
-      href: "https://www.kiwi.com/cz/pages/content/terms",
-      name: "Got to terms and condition",
-    },
-    {
-      name: "Reguest refund",
-      href: "#link",
-    },
-  ];
-  const buttonLabel = "https://www.kiwi.com/en/";
+  it("should have expected DOM output", () => {
+    const onClick = jest.fn();
+    const links = [
+      {
+        href: "https://www.kiwi.com/cz/pages/content/terms",
+        name: "Got to terms and condition",
+      },
+      {
+        name: "Go to something",
+        onClick,
+      },
+    ];
 
-  const component = shallow(<SkipLink links={links} buttonLabel={buttonLabel} />);
-
-  it("Should contain description block", () => {
-    expect(component.find("SkipLink__StyledLabel").exists()).toEqual(true);
-  });
-
-  it("Should have 2 Links", () => {
-    expect(component.find("SkipLink__StyledLink").at(1).exists()).toEqual(true);
+    render(<SkipLink links={links} label="https://www.kiwi.com/en/" />);
+    expect(screen.getByRole("link")).toHaveAttribute("href", links[0].href);
```

Comment:
The prior test weren't conforming to our testing conventions regarding [hardcoding values](https://github.com/kiwicom/orbit/blob/6c25d955f2cbca59010f859bcda4233c1918c128/.github/contribution/testing-conventions.md#pass-literal-props-directly), but I didn't request changes because they were used in a way where type checking would catch mistakes. However, I think accessing array elements and properties like this is leaves some chance of making a mistake, so please either hardcode values:
```suggestion
    expect(screen.getByRole("link")).toHaveAttribute("href", "Go to terms and conditions");
```
or save those values into variables:
```suggestion
    expect(screen.getByRole("link")).toHaveAttribute("href", termsAndConditions);
```